packages/gatsby-plugin-now/readme.md

```diff
@@ -1,24 +1,7 @@
 # gatsby-plugin-now
 
-This plugin generates [Now Routes](https://zeit.co/docs/v2/advanced/routes) for [redirects](https://www.gatsbyjs.org/docs/actions/#createRedirect) you configured for to your Gatsby project.
-
-### Usage
-
-1. Install the plugin:
+⚠️ The use of this plugin is deprecated. ZEIT Now supports Gatsby Redirects out-of-the-box and does not require the use a plugin.
```

Comment:
This is a reminder to actually run [`npm deprecate`](https://docs.npmjs.com/cli/deprecate) after we ship this PR to the stable branch.